pandas/core/strings.py

```diff
@@ -2093,6 +2093,11 @@ def __getitem__(self, key):
             return self.get(key)
 
     def __iter__(self):
+        warnings.warn(
+            "Columnar iteration over characters will be deprecated in future releases.",
+            DeprecationWarning,
```

Comment:
Can you raise a `FutureWarning` instead of a `DeprecationWarning`?

Reply:
✅